bedrock/settings/__init__.py

```diff
@@ -6,8 +6,8 @@
 
 from .base import *  # noqa
 
-if os.getenv('TRAVIS', False):
-    from .travis import *  # noqa
+if os.getenv('CIRCLECI', False):
+    from .circleci import *  # noqa
```

Comment:
I did. But I can probably go ahead and knock that out actually. I already did with the DB settings. I'll give it a go.